lib/_layouts/_base.njk

```diff
@@ -3,88 +3,123 @@
-  <link type="font/woff2" href="https://assets.nhs.uk/fonts/FrutigerLTW01-55Roman.woff2" rel="preload" as="font" crossorigin>
-  <link type="font/woff2" href="https://assets.nhs.uk/fonts/FrutigerLTW01-65Bold.woff2" rel="preload" as="font" crossorigin>
-
-  {% block headIcons %}
-  <!-- NHS Icons -->
-  <link rel="shortcut icon" href="{{ '/nhsuk-frontend/assets/favicons/favicon.ico' | url }}" type="image/x-icon">
-  <link rel="apple-touch-icon" href="{{ '/nhsuk-frontend/assets/favicons/apple-touch-icon-180x180.png' | url }}">
-  <link rel="mask-icon" href="{{ '/nhsuk-frontend/assets/favicons/favicon.svg' | url }}" color="#005eb8">
-  <link rel="icon" sizes="192x192" href="{{ '/nhsuk-frontend/assets/favicons/favicon-192x192.png' | url }}">
-  {% endblock %}
-
-  {% block msApplicationTiles %}
-  <meta name="msapplication-TileImage" content="{{ '/nhsuk-frontend/assets/favicons/mediumtile-144x144.png' | url }}">
-  <meta name="msapplication-TileColor" content="#005eb8">
-  <meta name="msapplication-square70x70logo" content="{{ '/nhsuk-frontend/assets/favicons/smalltile-70x70.png' | url }}">
-  <meta name="msapplication-square150x150logo" content="{{ '/nhsuk-frontend/assets/favicons/mediumtile-150x150.png' | url }}">
-  <meta name="msapplication-wide310x150logo" content="{{ '/nhsuk-frontend/assets/favicons/widetile-310x150.png' | url }}">
-  <meta name="msapplication-square310x310logo" content="{{ '/nhsuk-frontend/assets/favicons/largetile-310x310.png' | url }}">
-  {% endblock %}
-
-  {% block openGraph %}
-  <meta property="og:url" content="{{ page.url | url | absoluteUrl }}"/>
-  <meta property="og:site_name" content="nhsbsa.nhs.uk"/>
-  {% if description %}
-  <meta property="og:description" content="{{ description | markdown("inline") | striptags(true) }}">
-  {% endif %}
-  <meta property="og:type" content="website"/>
-  <meta property="og:locale" content="en_GB"/>
-  <meta property="og:image" content="{{ '/images/opengraph-image.png' | url | absoluteUrl }}">
-  <meta property="og:title" content="{% if title %}{{ title }} - {% endif %}{{ serviceName or productLongName or siteMeta.productLongName }} - {{ siteMeta.organisation }}">
-  {% endblock %}
-
-  {% block stylesheets %}
-  <link href="{{ '/stylesheets/application.css' | url }}" media="all" rel="stylesheet" type="text/css" />
-  {% endblock %}
-
-  {% block headEnd %}
-  {% endblock %}
-</head>
-{% endblock %}
-
-<body class="{{ bodyClasses }}" {%- for attribute, value in bodyAttributes %} {{attribute}}="{{value}}"{% endfor %}>
-
-  <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
-  
-  {% block bodyStart %}{% endblock %}
-
-  {% block skipLink %}
-    {{ skipLink({
+  <body class="{{ bodyClasses }}" {%- for attribute, value in bodyAttributes %} {{attribute}}="{{value}}"{% endfor %}>
+
+    <script>
+      document.body.className = (
+        (document.body.className)
```

Comment:
We dont have to do this here.
Wrap the cookie banner using the js css class instead like:
`<div class="js-enabled">`

Reply:
So this script is part of the standard default NHSUK template. It needs to be a script, because it only runs if JS is enabled. It helps with progressive enhancement so you can style and show JS only components when the 'js-enabled' class is present. If you just have a static `<div class="js-enabled">` you might end up with the incorrect css showing when JS is actually disabled. You can see it in the standard template for NHSUK here: [NHSUK base template](https://github.com/nhsuk/nhsuk-frontend/blob/main/app/_templates/layout.njk)

Reply:
If it's standard to check the document window then that's fine. I feel if something is properly wrapped with the correct css then it should work? Happy to close this thread.

Reply:
It depends because if you want something to be `display:block` when JS is enabled, but you're using a static 'js-enabled' class it'll also `display:block` when JS is disabled and that's not what you want.